recursio-client/UI/Menus/StartMenu.gd

```diff
@@ -246,11 +272,29 @@ func _on_game_result_received(_winning_player_id):
 	_return_to_game_room_lobby()
 
 
+func _on_tutorial_scenario_started() -> void:
+	_in_game = true
+
+
 func _on_tutorial_scenario_completed() -> void:
+	_in_game = false
 	_tutorial.show()
 
 
-func _on_tutorial_back_pressed() ->void:
+func _on_tutorial_back_pressed() -> void:
 	_tutorial.hide()
 	_start_menu_buttons.show()
 	_btn_play_tutorial.grab_focus()
+
+
+func _on_gameplay_menu_leave_pressed() -> void:
+	if not _in_game:
+		return
+	
+	if _world != null:
+		_return_to_game_room_lobby()
+	else:
```

Comment:
This feels unsafe (are we really definitely in a tutorial if the world is null and we're allegedly in-game?), might want to at least add an assert

Reply:
As `in_game` is only set when the world instantiated/freed (and same for the tutorial), it should be fine but will add them